src/core/utils/helpers/fetcherHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { store } from "../../store";
```

Comment:
I think you should rename the file to fetcher.ts. That is the naming convention we have used for other helpers.
Alternatively we need to rename all helper files 😈

Reply:
Sure